package.json

```diff
@@ -16,9 +16,23 @@
         "@reduxjs/toolkit": "^1.2.3",
         "@svgdotjs/svg.js": "^3.0.12",
         "@svgdotjs/svg.panzoom.js": "^2.1.2",
+        "@types/core-js": "^2.5.5",
+        "@types/dagre": "^0.7.48",
+        "@types/localized-countries": "^2.0.0",
+        "@types/lucene-escape-query": "^1.0.0",
         "@types/node": "^20.2.3",
+        "@types/plotly.js-basic-dist-min": "^2.12.1",
+        "@types/qs": "^6.9.7",
         "@types/react": "^18.2.7",
```

Comment:
need to update this to at least 18.2.8
this whole update was meant to make FunctionComponent work without any bug
we need at least typescript 5.1.* and @types/react 18.2.8+